examples/complete/fixtures.us-east-2.tfvars

```diff
@@ -1,10 +1,6 @@
-region = "us-east-2"
-
-namespace = "eg"
-
-environment = "ue2"
-
-stage = "test"
-
-name = "example"
-
+region          = "us-east-2"
+namespace       = "eg"
+environment     = "ue2"
+stage           = "test"
+create_iam_role = true
+enabled_rules   = ["arn:aws:inspector:us-east-2:646659390643:rulespackage/0-m8r61nnh"]
```

Comment:
Hrmmm.. tests should be able to run on other accounts (e.g. by contributors)

Reply:
This is actually an ARN of a rule from an AWS account.

See the [list of rules](https://docs.aws.amazon.com/inspector/latest/userguide/inspector_rules-arns.html) for possible ARNs. This is also documented in the `variables.tf` file.